doc/api/cli.md

```diff
@@ -585,6 +585,13 @@ added: v9.0.0
 Disables runtime checks for `async_hooks`. These will still be enabled
 dynamically when `async_hooks` is enabled.
 
+### `--no-force-napi-uncaught-exceptions-policy`
```

Comment:
We can name it with `node-api` instead of `napi` per https://github.com/nodejs/abi-stable-node/issues/420

Reply:
I was also thinking it might be something like

--force-napi-uncaught-exceptions-policy=false

with the default being the equivalent to

--force-napi-uncaught-exceptions-policy=true

If we find it ends up causing too much breakage then we'd change the default to false..